addons/im_livechat/static/tests/thread_readonly_tests.js

```diff
@@ -0,0 +1,52 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { Command } from "@mail/../tests/helpers/command";
+import { start } from "@mail/../tests/helpers/test_utils";
+import { contains, click } from "@web/../tests/utils";
+
+QUnit.module("Thread Readonly");
+
+QUnit.test("Rendering of readonly threads in Discuss", async () => {
```

Comment:
The test can be moved to mail as it is now a base feature related to being member.